Common Example code/Domains/Components/ComponentsViewController.swift

```diff
@@ -6,7 +6,7 @@
 
 import UIKit
 
-internal final class ComponentsViewController: UIViewController, Presenter {
+internal class ComponentsViewController: UIViewController, Presenter {
```

Comment:
why `final` removed ?

Reply:
removed the final qualifier to reuse the code in the deleted AdyenSPMUIHost target, but since the target is deleted, we can put back `final`.